ubuntu16@ubuntu:~/Downloads/gr-tutorial/python$ python qa_multiply_py_ff.py
E
======================================================================
ERROR: test_001_t (__main__.qa_multiply_py_ff)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "qa_multiply_py_ff.py", line 42, in test_001_t
    src = blocks.vector_source_f(src_data,2)
  File "/usr/lib/python2.7/dist-packages/gnuradio/blocks/blocks_swig1.py", line 2525, in make
    return _blocks_swig1.vector_source_f_make(*args, **kwargs)
TypeError: in method 'vector_source_f_make', argument 1 of type 'std::vector< float,std::allocator< float > > const &'

----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
